app/lib/dartdoc/backend.dart

```diff
@@ -24,6 +25,17 @@ import '../shared/versions.dart' as shared_versions;
 import 'models.dart';
 import 'storage_path.dart' as storage_path;
 
+const archiveFilePath = 'package.tar.gz';
+const blobFilePath = 'blob-data.gz';
+const blobIndexV1FilePath = 'index-v1.json';
+const buildLogFilePath = 'log.txt';
+const uploadedFilePaths = [
```

Comment:
Does this `uploadedFilePaths` need to exported?

Reply:
Yeah, the http handler doesn't try to serve these from the blob, rather it accesses them directly.